### ISSUE TITLE
Example 2.1.2 "Context with an object" is both wrong and misleading

### ISSUE THREAD

**Issue by zimeon** [w3c/activitystreams#524]
Please Indicate One: 
 * [x] Editorial
 * [ ] Question
 * [ ] Feedback
 * [ ] Blocking Issue
 * [ ] Non-Blocking Issue

Please Describe the Issue:

The AS2.0 [example 2.1.2 Context with an object](https://www.w3.org/TR/2017/REC-activitystreams-core-20170523/#h-example-2context) is both wrong and misleading. That is it wrong has already been pointed out in https://github.com/w3c/activitystreams/issues/510 but I'm creating a new issue because the problem is deeper than suggested there. The problem is that the suggested use of `@vocab` to map terms into the AS namespace (even if a `#` is added per https://github.com/w3c/activitystreams/issues/510) is not a substitute for specifying the AS context. For example, the `@vocab` will happily map `type` to `https://www.w3.org/ns/activitystreamstype` (or `https://www.w3.org/ns/activitystreams#type` if a `#` is added) but the [AS context](https://www.w3.org/ns/activitystreams.jsonld) actually maps `type` to `@type`

**Comment by gobengo** [comment]
I agree we should append '#' to the value of `@vocab` in example 2.1.2

**Comment by evanp** [comment]
I added the note to the Errata, and we can confirm with the larger group.

**Comment by evanp** [comment]
So, I think it's true that this is misleading, but with the corrected namespace it's not really wrong. I think the decision whether to remove the example entirely would be a good one to consider for the next version of AS2. A primer page on the topic might be useful; it might say something like "don't ever do this".

**Comment by trwnh** [comment]
It's not incorrect but it is redundant with the requirement that you MUST assume `https://www.w3.org/ns/activitystreams` even if it is missing within an AS2 document (application/activity+json)
